### DISCUSSION TITLE
backlight dimming

### DISCUSSION THREAD

**Discussion by psicho100pro** [OpenI6X/opentx#209] [Ideas]
I came across this note in the code.
I don't know what it's meant, because I don't know the design of other transmitters.
Does this mean that the firmware can't handle dimming, or is it that there is only a switching transistor on the PCB.
![2022-03-15_09h55_37](https://user-images.githubusercontent.com/97060304/158429866-b94e08ab-97da-490c-939a-1f1fef6e9902.png)
If we could output PWM to the (BL) MCU pin, then we could connect an external device that can do it.
![Screenshot 2022-03-15 at 10-05-37 32 49CZK 41% OFF DC 3 3V 3 7V 5V LED Driver 30 1500MA Constant Current Adjustable Module ](https://user-images.githubusercontent.com/97060304/158430124-33363008-6a75-4ba2-be07-6ecb239eac08.png)
According to the picture, only R19 would be removed on the PCB. The output from the module is connected to the LED pads 1 and 2. We also have Vbat + BL + GND for the module.
![2022-03-15_08h58_17](https://user-images.githubusercontent.com/97060304/158430245-09ad6873-ef16-4027-bdcf-0a9b4ff662e3.png)
What do you think about it?

**Comment by ajjjjjjjj** [comment]
It is possible by creating timer irq handler and toggling BL pin manually. This is sub optimal but do not require any H/W changes.
